instrumentation/sinatra/test/opentelemetry/instrumentation/sinatra_test.rb

```diff
@@ -156,6 +157,7 @@
         'http.scheme' => 'http',
         'http.target' => '/error'
       )
+      _(exporter.finished_spans.first.events.first.name).must_equal('exception')
```

Comment:
Non-blocking suggesting to make this a little more functional.  

```suggestion
      _(exporter.finished_spans.flat_map(&:events).map(&:name)).must_equal(['exception'])
```